### DISCUSSION TITLE
v2 API - How long does an OTP token last before expiring?

### DISCUSSION THREAD

**Discussion by cliveportman** [supabase/supabase#9352] [Questions] [answered]
I seem to be timing out really quickly.

**Comment by GaryAustin1** [comment, marked as answer]
You set the time under authentication settings in the dashboard for either email or phone providers.  Looks like email defaults to 24 hours and phone to 60 seconds.

**Reply by cliveportman** [reply]
My email is on 60 seconds, lol. Thanks.
